### DISCUSSION TITLE
[Question] How do i enable users to copy text from an App?

### DISCUSSION THREAD

**Discussion by JonasMedu** [flet-dev/flet#1290] [Q&A] [answered]
### Question

Given this very simple example App (see code below).
What do i have to change, for the user to be able to mark the text "Hello, world!" with the users mouse and copy it to the clipboard?

### Code sample

```python
import flet as ft

def main(page: ft.Page):
    t = ft.Text(value="Hello, world!", color="green")
    page.controls.append(t)
    page.update()

ft.app(target=main)
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by DavMelchi** [comment, marked as answer]
Text has selectable property

`t = ft.Text(value="Hello, world!", color="green",selectable=True )`

https://flet.dev/docs/controls/text#selectable

**Reply by JonasMedu** [reply]
awesome :1st_place_medal:  thanks 
Is there any comfortable setting to set this globally as default?

**Reply by DavMelchi** [reply]
No global option available as I know.

**Reply by ndonkoHenri** [reply]
> Is there any comfortable setting to set this globally as default?

@FeodorFitsner is this perhaps possible? Something like `page.text_selectable` to make text selection global.
